assets/js/googlesitekit/datastore/user/user-input-settings.js

```diff
@@ -202,7 +202,10 @@ export const baseControls = {
 				const settings =
 					registry.select( CORE_USER ).getUserInputSettings() || {};
 
-				settings[ settingID ] = { values };
+				settings[ settingID ] = {
+					...( ( settings || {} )[ settingID ] || {} ),
```

Comment:
It's a minor point, but I'd suggest using optional chaining here to make the line slightly shorter and I think easier to read:

```suggestion
					...( settings?.[ settingID ] || {} ),
```

Reply:
Good call. Updated, thank you!